### PR TITLE
feat(docs): add normalized option to CM docs

### PR DESCRIPTION
**Pull request by shcheklein**
Per https://github.com/iterative/dvclive/pull/538

------------

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/dvclive/live/log_sklearn_plot.md [hunk 1]
```diff
@@ -192,18 +192,23 @@ $ dvc plots show 'dvclive/plots/sklearn/roc.json' \
...
 
-- `**kwargs` - additional arguments to be passed to the internal scikit-learn
-  function being called
+- `**kwargs` - additional arguments to tune the result. Arguments are passed to
+  the scikit-learn function (e.g. `drop_intermediate=True` for the
+  [`roc`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
+  type). Plus extra arguments supported by the type of a plot are:
+
+  - `normalized` - _default_: `False`. `confusion_matrix` with values normalized
+    to `<0, 1>` range.
```
Comment (daavoo) [inline content/docs/dvclive/live/log_sklearn_plot.md:211]: Didn't want to block the code P.R. as it is not critical but this highlights what I consider a problem which is that `confusion_matrix` should not really be part of `log_sklearn_plot`, as it doesn't really call `sklearn` under the hood.

**Review by daavoo — APPROVED** [review]
Would be maybe good to add a mention in the tabs above

File: content/docs/dvclive/live/log_sklearn_plot.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/dvclive/live/log_sklearn_plot.md:211]: Should we move confusion matrix to https://github.com/iterative/dvclive/issues/271 once we implement that? We could make a new issue or note it there.

Comment (daavoo) [reply content/docs/dvclive/live/log_sklearn_plot.md:211]: > Should we move confusion matrix to https://github.com/iterative/dvclive/issues/271 once we implement that? We could make a new issue or note it there.

Yep. Would probably need to keep it hidden but working until the next major release.

I have just started working on #271 , it looks simple enough but useful to showcase now that we are pushing for DVCLive more
